sycl/doc/extensions/proposed/sycl_ext_intel_cache_controls.asciidoc

```diff
@@ -98,85 +98,210 @@ Below is a list of new compile-time constant properties supported with
 ```c++
 namespace sycl::ext::intel::experimental {
 
-enum class cache_level : /*unspecified*/ {
+enum class level : /*unspecified*/ {
     L1,
     L2,
     L3,
     L4,
 };
 
 enum class cache_control_read_type : /* unspecified */ {
-    cached,
     uncached,
+    cached,
     streaming,
     invalidate_after_read,
-    const_cached
+    const_cached,
```

Comment:
What is the value of adding a coma here?

Reply:
No real value. Can be removed. Some enums in the compiler tend to use dangling commas.